certbot-compatibility-test/certbot_compatibility_test/util.py

```diff
@@ -26,12 +26,12 @@ def create_le_config(parent_dir):
     config = copy.deepcopy(constants.CLI_DEFAULTS)
 
     le_dir = os.path.join(parent_dir, "certbot")
-    config["config_dir"] = os.path.join(le_dir, "config")
-    config["work_dir"] = os.path.join(le_dir, "work")
-    config["logs_dir"] = os.path.join(le_dir, "logs_dir")
-    os.makedirs(config["config_dir"])
-    os.mkdir(config["work_dir"])
-    os.mkdir(config["logs_dir"])
+    os.mkdir(le_dir)
+    for dir_name in ("config", "logs", "work"):
+        full_path = os.path.join(le_dir, dir_name)
+        os.mkdir(full_path)
+        full_name = dir_name + "_dir"
+        config[full_name] = full_path
```

Comment:
this changes `config` to `config_dir`; that's definitely ok?

Reply:
I'm not sure what you mean. The only change in the end result here should be the basename of the log directory is `logs` rather than `logs_dir`.

Overall change was caused by `mypy` being unsure about the type returned from `config[name]`, even if we set it earlier in the function.

Reply:
yes, that is the direction I meant. is that ok to change the name.

Reply:
Should be. Only references to `logs_dir` in this package is in this function and Certbot doesn't expect the directory to end in `logs_dir`.